components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -354,6 +354,7 @@ be lost if SCREAM_HACK_XML is not enabled.
       >
         false
       </extra_clnsky_diag>
+      <do_subcol_sampling type="logical">true</do_subcol_sampling>
```

Comment:
Would you mind adding a `doc="blah"` string explaining what this parameter does?

Reply:
Done.